src/UserSettingsStore.js

```diff
@@ -180,33 +190,30 @@ export default {
         localStorage.setItem('mx_local_settings', JSON.stringify(settings));
     },
 
-    getFeatureById(feature: string) {
-        for (let i = 0; i < this.LABS_FEATURES.length; i++) {
-            const f = this.LABS_FEATURES[i];
-            if (f.id === feature) {
-                return f;
-            }
-        }
-        return null;
-    },
-
     isFeatureEnabled: function(featureId: string): boolean {
-        // Disable labs for guests.
-        if (MatrixClientPeg.get().isGuest()) return false;
+        const featuresConfig = SdkConfig.get()['features'];
```

Comment:
"features" doesn't strike me as particularly grep-able. How about `featuresConfig`?

Reply:
This is what I put in the doc, so we should go back & discuss there, although featuresConfig seems a bit redundant given it's in a config file

Reply:
`featuresDeployState`? `featuresControl`?